.editorconfig

```diff
@@ -0,0 +1,123 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
```

Comment:
Actually, I suppose the closest rules take precedence, so if this editorconfig is naming all rules then any parent directory .editorconfig would have no effect- as this would override.

Reply:
> it will ignore parent .editorconfigs

Yes... But this is intended, actually. In my mind, this file should be the parent one. Our C# code base should not have another rule set with higher level.

Reply:
Agreed that our code base shouldn't- but I think a developer should be able to on their machine. The rules defined in this file will override any parent rules anyway.

I'm thinking like this, for every rule:
* If we think it's important for our code base, it should be in this file. 
* If we think it's not important for our code base, we shouldn't care if a parent `.editorconfig` is doing something to it.